packages/fmg-core/bin/test.sh

```diff
@@ -0,0 +1,3 @@
+cd built
```

Comment:
What happens if the `built` directory doesn't exist here?

Reply:
It wouldn't work! I guess `truffle test` would ignore the typescript files, and we could end up with a false positive test result.

Right now, `npm build` ensures that `built` exists, so it's not a problem. But we should make sure that `test.sh` should exit with non-zero error code if `built` doesn't exist.